profiles/common/modules/features/nexteuropa_geofield/nexteuropa_geofield.module

```diff
@@ -317,3 +317,14 @@ function _nexteuropa_geofield_get_geojson($node) {
   print $geojson_data;
   drupal_exit();
 }
+
+/**
+ * Implements hook_node_validate().
+ */
+function nexteuropa_geofield_node_validate($node, $form, &$form_state) {
+  // When the node form is not valid, custom javascripts must be loaded.
+  $field_instance = $form_state['field']['field_geo'][LANGUAGE_NONE];
```

Comment:
Not correct : this is the name of the field.
I had the following message : 
Notice: Undefined index: field_geo in nexteuropa_geofield_node_validate() (line 326 of /ec/dev/server/fpfis/webroot/sources/senseca/platform-dev/profiles/common/modules/features/nexteuropa_geofield/nexteuropa_geofield.module).
Notice: Undefined property: stdClass::$field_geo in nexteuropa_geofield_node_validate() (line 328 of /ec/dev/server/fpfis/webroot/sources/senseca/platform-dev/profiles/common/modules/features/nexteuropa_geofield/nexteuropa_geofield.module).